test/lib/ufe/testContextOps.py

```diff
@@ -358,12 +359,26 @@ def testAddNewPrimWithDelete(self):
         self.assertEqual(len(proxyShapehier.children()), 1)
 
         # Using UFE, delete this new prim (which doesn't actually delete it but
-        # instead makes it inactive).
+        # instead makes it inactive). In Maya 2023 and greater, delete really
+        # deletes so deactivate instead to make the test act the same for all
+        # versions.
         cmds.pickWalk(d='down')
-        cmds.delete()
-
-        # The proxy shape should now have no UFE child items (since we skip inactive).
-        self.assertFalse(proxyShapehier.hasChildren())
+        if mayaUtils.mayaMajorVersion() >= 2023:
+            ufeItem = ufe.GlobalSelection.get().front()
+            item = usdUtils.getPrimFromSceneItem(ufeItem)
+            item.SetActive(False)
+        else:
+            cmds.delete()
+
+        # The proxy shape should now have no UFE child items (since we skip inactive
+        # when returning the children list) but hasChildren still reports true in
+        # UFE version before 0.4.4 for inactive to allow the caller to do conditional
+        # inactive filtering, so we test that hasChildren is true for those versions.
+        ufeVersion = ufeUtils.fullVersion()
+        if (0, 4, 4) <= ufeVersion < (1, 0, 0):
+            self.assertFalse(proxyShapehier.hasChildren())
+        else:
+            self.assertTrue(proxyShapehier.hasChildren())
```

Comment:
We have been doing this in a different way that allows us to find and remove these checks at a later date when a new Maya PR ships. Once the Maya PR ships with this new Ufe 0.4.x version we don't need a test on the specific Ufe version anymore - just a check against Ufe v4 is okay. So to find them we always use either the existing `ufeFeatureSetVersion` function or `UFE_PREVIEW_VERSION_NUM` and fix/remove them. So I think your test should be something like:
`if (os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') > '4004'):`
This cmake variable is also set as an env var for all these Ufe tests.